tools/rosconsole/src/rosconsole/rosconsole.cpp

```diff
@@ -425,6 +436,19 @@ void initialize()
     backend::function_notifyLoggerLevelsChanged = notifyLoggerLevelsChanged;
     backend::function_print = _print;
 
+    std::string line_buffered;
+    if (get_environment_variable(line_buffered, "ROSCONSOLE_STDOUT_LINE_BUFFERED"))
+    {
+      if (line_buffered == "1") {
+        g_force_stdout_line_buffered = true;
+      } else {
+        if (line_buffered != "0") {
```

Comment:
If you are ok with it I can just apply the change to your branch (thanks to the latest features in GitHub :wink:).